src/System.Net.Sockets/tests/FunctionalTests/UnixDomainSocketTest.cs

```diff
@@ -1,249 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
```

Comment:
To avoid needing to pollute the test project with the resources and native shim dependency. I could put it back with those, as back a copy of the original sample code that didn't have them, or leave it as is. I went with this figuring we're getting much more coverage from named pipes using it and this wasn't adding much beyond that.